snuba/query/snql/__init__.py

```diff
@@ -29,19 +35,23 @@
     visit_parameters_list,
     visit_quoted_literal,
 )
-from typing import Sequence
-
 
 snql_grammar = Grammar(
     fr"""
-    query_exp             = match_clause where_clause? collect_clause? group_by_clause? having_clause? order_by_clause?
+    query_exp             = match_clause where_clause collect_clause group_by_clause? having_clause? order_by_clause?
```

Comment:
How come the where and collect clauses are now mandatory? Wouldn't a query without these still be valid?

Reply:
I think that makes sense - I gave it some thought and compared with Cypher query language as well - it seems to me that a query would still be valid without a WHERE or COLLECT?